JSONs/RBI_laws_failed_txn.json

```diff
@@ -0,0 +1,113 @@
+[{
+        "qId": 1,
+        "Questions": "What is your PINCODE?",
+        "Parameters": "area_of_user",
+        "Condition": "rural",
+        "LegalTrainingMaterial": "\nIf the pincode belongs to rural \/ semi-urban district, then refer to the following:\n\u201cBank is well aware that I belong to a rural \/ semi-urban area where the financial literacy is assumed to be low, and customers from these areas are more susceptible to frauds. There is a heightened duty casted on the banks to implement robust technological solutions to protect the vulnerable customers especially those from rural \/ semi-urban areas from financial frauds. The bank should be called upon to demonstrate what additional technology and security parameters the bank has put in place to protect the customers from rural \/ semi-urban areas from the financial frauds.\u201d\n"
```

Comment:
remove this special characters from every paragraph.
replace it with normal "double quotes"